extensions/wikia/FilePage/FilePage.setup.php

```diff
@@ -43,4 +43,11 @@
 $wgHooks['GlobalUsageImagePageWikiLink'][] = 'FilePageHooks::onGlobalUsageImagePageWikiLink';
 $wgHooks['GlobalUsageLinksUpdateComplete'][] = 'FilePageHooks::onGlobalUsageLinksUpdateComplete';
 $wgHooks['BeforePageDisplay'][] = 'FilePageHooks::onBeforePageDisplay';
-$wgHooks['WikiaMobileAssetsPackages'][] = 'FilePageHooks::onWikiaMobileAssetsPackages';
\ No newline at end of file
+$wgHooks['WikiaMobileAssetsPackages'][] = 'FilePageHooks::onWikiaMobileAssetsPackages';
+if ( !empty( $wgRedirectFilePagesForAnons ) ) {
```

Comment:
OK, this was mostly to prevent purging cache keys on wikis where nothing would be cached for file redirects, but it should be relatively harmless to re-enable them. So, I'm OK with removing the feature flag here. :)